src/js/game/LevelMVC/LevelView.js

```diff
@@ -191,6 +194,18 @@ module.exports = class LevelView {
       "pistonArmRight": ["blocks", "piston_arm_right", -26, -13],
       "pistonArmUp": ["blocks", "piston_arm_up", -26, -13],
       "pistonArmDown": ["blocks", "piston_arm_down", -26, -13],
+
+      "cactus": ["blocks", "cactus", -13, 0],
+      "dead_bush": ["blocks", "dead_bush", -13, 0],
+      "glowstone": ["blocks", "glowstone", -13, 0],
+      "grass_path": ["blocks", "grass_path", -13, 0],
+      "ice": ["blocks", "ice", -13, 0],
+      "netherrack": ["blocks", "netherrack", -13, 0],
+      "nether_brick": ["blocks", "nether_brick", -13, 0],
+      "quartz_ore": ["blocks", "quartz_ore", -13, 0],
+      "snow": ["blocks", "snow", -13, 0],
+      "snowy_grass": ["blocks", "snowy_grass", -13, 0],
+      "top_snow": ["blocks", "top_snow", -13, 0],
```

Comment:
I'm pretty sure at least some of these shouldn't be actual blocks, much like the "tallGrass" asset

Reply:
I'm actually of the opinion that -everything- in blocks[] should be a block... It would appear that the top half of everything is listed as a block (done in year 1) and then whoever did year 2 decided to start labeling things as "not blocks". 

Tall grass for instance, is basically the same as flowerRose (in that it's only meant to be used in the decorationPlane). flowerRose, as with all the flowers, are also given the key "blocks". 

The one sort of rational exception, in my opinion are entries with added behavior. Like tree leaves and doors, as they animate.

None of the additional entries in this PR, with the exception of "leavesSpruceSnowy" animate in any way, and are thus categorized as blocks like all the other non-animated entries.

Reply:
Huh, right you are. I had a misunderstanding on what it means to be not-a-block.

Thanks for the solid explanation!